migrations/12_SCHEME-REP-Redeem_dappcon_cards.js

```diff
@@ -1,15 +1,47 @@
-/* global artifacts */
+/* global artifacts, web3 */
 /* eslint no-undef: "error" */
 
+const DxRedeemDappconCards = artifacts.require('DxRedeemDappconCards')
+const DxAvatar = artifacts.require('DxAvatar')
+const DxController = artifacts.require('DxController')
+
+const dateUtil = require('../src/helpers/dateUtil')
+
+const registerScheme = require('./helpers/registerScheme')
+
+const {
+  reputationReward,
+  redeemEnableTime: redeemTime
+} = require('../src/config/schemes/old/fixreputationallocationparams.json')
+
 module.exports = async function (deployer) {
-  console.log('TODO: Deploy RedeemDappconCards that inherits from FixedReputationAllocation')
-  console.log('Configure RedeemDappconCards')
-  // TODO: Check permissions OLD_dao_migration.js
-  console.log('controller.registerScheme(_schemes[i], _params[i], _permissions[i],address(_avatar));')
-
-  // await deployer.deploy(FixedReputationAllocation,
-  //   AvatarInst.address,
-  //   fixedReputationAllocationParamsJson.reputationReward,
-  //   options)
-  // var fixedReputationAllocationInst = await FixedReputationAllocation.deployed()
+  const dxAvatar = await DxAvatar.deployed()
+  const dxController = await DxController.deployed()
+
+  console.log('Deploy DxRedeemDappconCards that inherits from FixedReputationAllocation')
+  const dxRedeemDappconCards = await deployer.deploy(DxRedeemDappconCards)
+
+  console.log('Configure DxRedeemDappconCards')
+
+  // TODO: have real times in config, for now current time + from config
+  // to ensure times are in the future
+  const redeemEnableTime = new Date(Date.now() + redeemTime * 1000)
```

Comment:
Here we should use an approach like with the locking period params. Loading the dates for the central conf `conf/timePeriods`. By the way we now have more info about the time periods, we could use the same naming convention so it's more clear:

![image](https://user-images.githubusercontent.com/2352112/48483461-cf9d1100-e813-11e8-9617-11e10c7fc2d8.png)